main.py

```diff
@@ -1,10 +1,44 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player_name, player_data in players.items():
+        guild_data = player_data.get("guild")
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                description=guild_data["description"]
+            )
+        else:
+            guild = None
+
+        race_data = player_data.get("race")
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
```

Comment:
Simplify this to directly retrieve `player_data["race"]["name"]` without declaring `race_data` variable.